virion.yml

```diff
@@ -1,5 +1,5 @@
 name: libFormAPI
 author: jojoe77777
 api: 4.0.0
-version: 2.1.1
+version: [4.0.0, 5.0.0]
```

Comment:
This is the wrong field, it's `api` that should be modified (good catch @PJZ9n )

Reply:
hahaha, I can't believe I made this